app/graph/types/user_type.rb

```diff
@@ -6,4 +6,9 @@
   field :first_name, !types.String
   field :last_name, types.String
   field :email, !types.String
+
+  field :posts, types[PostType]
+  field :wall, types[PostType]
+  field :comments, types[CommentType]
+  field :likes, types[LikeType]
```

Comment:
@DerekStride This field wont resolver properly because it expects two ids

```
  field :like, LikeType do
    argument :user_id, !types.ID
    argument :content_id, !types.ID

    resolve -> (_, args, _) { Like.find_by(user_id: args[:user_id], content_id: args[:content_id]) }
  end
```

Reply:
Is there a way in graphql to allow 'overloaded' types, similar to overloaded methods in OO programming. Does that even make sense.

Reply:
met with derek IRL, we fixed the issues.